jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -253,23 +255,21 @@ class ResourceSelector<T> private constructor(
 
         fun disableAutomaticSorting(): ResourceBuilderOptions<T> = also { it.sortOnLoad = false }
 
-        fun awsConnection(awsConnection: AwsConnection): ResourceBuilderOptions<T> = awsConnection { awsConnection }
+        fun activeAwsConnection(project: Project): ResourceBuilderOptions<T> = awsConnection { AwsConnectionManager.getInstance(project).connectionSettings() }
```

Comment:
Would prefer this also be named `awsConnection`. We grab the connection at that point in time, so it doesn't change like might be implied by `active`. Also makes the API make more sense

Reply:
renamed